docs/en_US/ResearchBlog/NASComparison.md

```diff
@@ -0,0 +1,81 @@
+# NAS Algorithms Comparison
+Train and Compare NAS models including Autokeras, DARTS, ENAS and NAO.
+
+Their source code link is as below:
+
+- Autokeras: [https://github.com/jhfjhfj1/autokeras](https://github.com/jhfjhfj1/autokeras)
+
+- DARTS: [https://github.com/quark0/darts](https://github.com/quark0/darts)
+
+- ENAS: [https://github.com/melodyguan/enas](https://github.com/melodyguan/enas)
+
+- NAO: [https://github.com/renqianluo/NAO](https://github.com/renqianluo/NAO)
+
+## Experiment Description
+
+To avoid over-fitting in **CIFAR-10** (I hate this dataset) , we also compare the models in the other five datasets including Fashion-MNIST, CIFAR-100, OUI-Adience-Age, ImageNet-10-1 (subset of ImageNet), ImageNet-10-2 (another subset of ImageNet). We just sample a subset with 10 different labels from ImageNet to make ImageNet-10-1 or ImageNet-10-2.
```

Comment:
As this is not a personal site, let's remove 
> (I hate this dataset)

Reply:
Thank you for your advice.